test/unit/lib/pageTitles.js

```diff
@@ -0,0 +1,63 @@
+const chai = require('chai');
+const pageTitles = require('../../../app/lib/pageTitles');
+
+const expect = chai.expect;
+
+describe('displayTitles', () => {
+  describe('search', () => {
+    it('should return \'Find a pharmacy\' when no error message', () => {
+      const location = '';
+      const errorMessage = undefined;
+      const result = pageTitles.search(location, errorMessage);
+      expect(result).to.equal('Find a pharmacy');
+    });
+
+    it('should return \'Find a pharmacy - Enter a town, city or postcode\' when error message, and no location provided', () => {
+      const location = '';
+      const errorMessage = 'Please enter a location';
+      const result = pageTitles.search(location, errorMessage);
+      expect(result).to.equal('Find a pharmacy - Enter a town, city or postcode, or use your location');
+    });
+
+    it('should return \'Find a pharmacy - We can\'t find the postcode ...\' with uppercase location when error message, and location provided', () => {
+      const location = 'notapostcode';
+      const errorMessage = 'Please enter a location';
+      const result = pageTitles.search(location, errorMessage);
+      expect(result).to.equal('Find a pharmacy - We can\'t find the postcode \'NOTAPOSTCODE\'');
+    });
+  });
+
+  describe('disambiguation', () => {
+    it('should return \'Find a pharmacy - We can\'t find ...\' when no places provided', () => {
+      const location = 'no such place';
+      const places = [];
+      const result = pageTitles.disambiguation(location, places);
+      expect(result).to.equal('Find a pharmacy - We can\'t find \'no such place\'');
+    });
+
+    it('should return \'Find a pharmacy - places that match ...\' when places provided', () => {
+      const location = 'Leeds';
+      const places = [{ name: 'Leeds1' }, { name: 'Leeds2' }];
+      const result = pageTitles.disambiguation(location, places);
+      expect(result).to.equal('Find a pharmacy - Places that match \'Leeds\'');
+    });
+  });
+
+  describe('results', () => {
```

Comment:
There is a test missing for when there are zero openServices but there are some nearbyServices.

Reply:
I've added one for open services but no nearby services too.